src/homework21/person/PersonSort.java

```diff
@@ -0,0 +1,81 @@
+package homework21.person;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Scanner;
+
+public class PersonSort {
+    //"C:\\Users\\User\\IdeaProjects\\text\\temp.txt";
+    public static String filePath;
```

Comment:
This also shouldn't be class-level variable.
You should move it into `main` method and pass as a parameter to `writeFile` as well